tests/test_correlatednoise.py

```diff
@@ -36,14 +36,6 @@
 smallim_size = 16 # size of image when we test correlated noise properties using small inputs
 smallim_size_odd = 17 # odd-sized version of the above for odd/even relevant tests (e.g. draw)
 largeim_size = 12 * smallim_size # ditto, but when we need a larger image
-xlargeim_size =long(np.ceil(1.41421356 * largeim_size)) # sometimes, for precision tests, we 
-                                                        # need a very large image that will 
-                                                        # fit a large image within it, even if 
-                                                        # rotated
-
-# Decimals for comparison (one for fine detail, another for comparing stochastic quantities)
-decimal_approx = 2
-decimal_precise = 7
```

Comment:
Well it's up to you, as I will not be active from now on, but that's not how I worked.  Some of these tests take a long time to run, and share similar properties, and I found it helpful to be able to ramp up accuracy across the board as much of the *real* output of these tests when building the code was plots, and these changed very informatively as accuracy was increased.  You don't work that way, fine, but you are taking an executive decision to move to your own preferred approach here and I wanted to understand why, and flag it.  I have no idea why global variables aren't a sensible idea in a test situation rather than buried deep in the body of the code.